<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>371</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="principalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>50</number>
     </property>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="lista1"/>
     </item>
     <item row="1" column="3">
      <layout class="QVBoxLayout" name="lista4"/>
     </item>
     <item row="1" column="4">
      <layout class="QVBoxLayout" name="lista5"/>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="lista2"/>
     </item>
     <item row="1" column="2">
      <layout class="QVBoxLayout" name="lista3"/>
     </item>
     <item row="0" column="7">
      <layout class="QVBoxLayout" name="completo4"/>
     </item>
     <item row="0" column="4">
      <layout class="QVBoxLayout" name="completo2"/>
     </item>
     <item row="0" column="6">
      <layout class="QVBoxLayout" name="completo3"/>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="masoOculto"/>
     </item>
     <item row="1" column="6">
      <layout class="QVBoxLayout" name="lista6"/>
     </item>
     <item row="1" column="7">
      <layout class="QVBoxLayout" name="lista7"/>
     </item>
     <item row="0" column="3">
      <layout class="QVBoxLayout" name="completo1"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>df</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>724</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
